lodash.js

```diff
@@ -9833,6 +9833,25 @@
     }
 
     /**
+     * The abstract operation toLength converts its argument to an integer
+     * suitable for use as the length of an array-like object.
+     *
+     * **Note:** This function is based on [`ToLength`](http://ecma-international.org/ecma-262/6.0/#sec-tolength).
+     *
+     * @static
+     * @memberOf _
+     * @category Lang
+     * @param {*} value The object to be converted to a length.
+     * @param {number} [length=MAX_SAFE_INTEGER] The upper bounds of a valid length.
+     * @return {number} If len <= 0 then 0 else if len is +INFINITY then 2^53-1
+     *                  else min(len, 2^53-1).
+     */
+    function toLength(value, length) {
```

Comment:
Since this isn't used internally you can remove the `length` param and just ensure the value is clamped to the `MAX_ARRAY_LENGTH` and greater than `-1`.